### PR TITLE
Add: control timeout while serving HTTP for RPC.

### PR DESCRIPTION
**Pull request by jiusanzhou**
Access timeout of RPC by HTTP's header `Timeout`.
The default timeout is 10 second.

### CODE REVIEW

**Comment by asim** [comment]
Is the default timeout for RPC actually 10 seconds though?

**Comment by jiusanzhou** [comment]
> Is the default timeout for RPC actually 10 seconds though?

Sorry Asim, I was just set 10 seconds as the default timeout in my own business code.
I have changing the code with the default timeout following the client.DefaultRequestTimeout.
Plz review PR again, and decide whether you can merge the PR.

**Comment by asim** [comment]
This should really be

```
       var opts []client.CallOption

	timeout, _ := strconv.Atoi(r.Header.Get("Timeout"))
        // set timeout
	if timeout > 0 {
		opts = append(opts, client.WithRequestTimeout(time.Duration(timeout) * time.Second))
	}

 	// remote call
	if len(address) > 0 {
                opts = append(opts, client.WithAddress(address))
        }

	err = (*cmd.DefaultOptions().Client).Call(ctx, req, &response, opts...)
```

File: internal/handler/rpc.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/micro/go-micro/cmd"
 	"github.com/micro/go-micro/errors"
 	"github.com/micro/micro/internal/helper"
+	"time"
```
Comment (asim) [inline internal/handler/rpc.go]: See the styling. This should be imported with the other stdlib imports.

**Pull request merged**